lib/u3d/cache.rb

```diff
@@ -41,6 +41,9 @@ class Cache
     # Currently set to 24h
     CACHE_LIFE = 60 * 60 * 24
 
+    # Cache version so we know which global version to fetch
+    CACHE_VERSION = 0
```

Comment:
why not 1?

Reply:
And I am not sure I want a dependency like that. If it's only used in the URL, it's not up to u3d to dictate it, but `unities` to define it.